lib-extra/build.gradle

```diff
@@ -85,6 +85,26 @@ p2deps {
 		p2repo 'https://download.eclipse.org/tools/cdt/releases/10.7/'
 		install 'org.eclipse.cdt.core'
 	}
+
+	/*
+	 * JDT core manipulation required for clean-up base interfaces and import sorting
+	 * It depends on JDT core, which is required for formatting.
+	 */
+	/**compile("org.eclipse.jdt:org.eclipse.jdt.core.manipulation:${VER_ECLIPSE_JDT_CORE_MANIPULATION}") {
```

Comment:
@nedtwigg This was in  the old build.gradle. I suppose it has to be converted into things like:

	public static EquoBasedStepBuilder createBuilder(Provisioner provisioner) {
		return new EquoBasedStepBuilder(NAME, provisioner, EclipseJdtCleanUpStep::apply) {
			@Override
			protected P2Model model(String version) {
				var model = new P2Model();
				addPlatformRepo(model, version);
				model.getInstall().add("org.eclipse.jdt.core");
				return model;
			}

Reply:
That is correct.